app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ArticlesController < ApplicationController
+  
+  http_basic_authenticate_with name: "user", password: "secret", except: [:index, :show]
+
+  def index
+    @articles = Article.all
+  end
+
+  def show
+    @article = Article.find(params[:id])
+  end
+
+  def new
+    @article = Article.new
+  end
+
+  def create
+    # @article = Article.new(title: "Amazing Animals", body: "Show of top 10 Animals"title: "Amazing Animals", body: "Show of top 10 Animals")
```

Comment:
Remember always to remove commented code.

Reply:
Did it !